CONTRIBUTING.md

```diff
@@ -20,22 +20,41 @@ PRs are encouraged, but we suggest starting with a Feature Request to temperatur
+
+After a pull request is submitted, a single approval is required to merge it.
+
 ### Auto-releasing and commit conventions
 
-A new version of this package will be automatically published whenever there is a merge to the `main` branch. Specifically, new GitHub releases and tags will be created, and a new NPM package version will be published. The release version increment type is derived from the commits that were bundled in the merge to `main`, using [semantic-release commit message conventions](https://github.com/semantic-release/semantic-release#commit-message-format).
+A new version of this package will be automatically published whenever there is a merge to the `main` branch. Specifically, new GitHub releases and tags will be created, and a new NPM package version will be published. The release version increment type is derived from the format of the commit messages that were bundled in the merge to `main`, using [semantic-release commit message conventions](https://github.com/semantic-release/semantic-release#commit-message-format).
 
 The table below shows example commits and the resulting release type:
 
 <!-- prettier-ignore-start -->
 | Commit message                                                                                                                                                                                   | Release type                                                                                                    |
 | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | --------------------------------------------------------------------------------------------------------------- |
-| `fix(pencil): stop graphite breaking when too much pressure applied`                                                                                                                             | ~~Patch~~ Fix Release                                                                                           |
-| `feat(pencil): add 'graphiteWidth' option`                                                                                                                                                       | ~~Minor~~ Feature Release                                                                                       |
+| `docs(pencil): describe scribble feature` | No Release |
```

Comment:
@richardhuaaa I have a question about these commit messages - and it is not specific to your PR - but seeing this here makes me wonder what the `(pencil)` part of the message means. =) I don't usually include it when I'm using the `docs:` message. Do you happen to know the significance of it?